app/lib/features/chat_ng/rooms_list/widgets/rooms_list.dart

```diff
@@ -0,0 +1,263 @@
+import 'package:acter/common/themes/colors/color_scheme.dart';
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter/common/widgets/acter_search_widget.dart';
+import 'package:acter/common/widgets/plus_icon_widget.dart';
+import 'package:acter/features/chat/models/room_list_filter_state/room_list_filter_state.dart';
+import 'package:acter/features/chat/providers/room_list_filter_provider.dart';
+import 'package:acter/features/chat_ng/globals.dart';
+import 'package:acter/features/chat_ng/rooms_list/widgets/chats_list.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:acter/l10n/generated/l10n.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:go_router/go_router.dart';
+
+typedef RoomSelectAction = Function(String);
+
+class RoomsListNGWidget extends ConsumerStatefulWidget {
```

Comment:
assuming this been just moved from elsewhere and not logically modified ?

Reply:
move and renames, correct.